src/components/AttachmentCarousel/index.js

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import * as Expensicons from '../Icon/Expensicons';
+import styles from '../../styles/styles';
+import themeColors from '../../styles/themes/default';
+import CarouselActions from './CarouselActions';
+import Button from '../Button';
+import * as ReportActionsUtils from '../../libs/ReportActionsUtils';
+import * as Report from '../../libs/actions/Report';
+
+import AttachmentView from '../AttachmentView';
+import addEncryptedAuthTokenToURL from '../../libs/addEncryptedAuthTokenToURL';
+import canUseTouchScreen from '../../libs/canUseTouchscreen';
```

Comment:
I think this should be. The app fails to build right now

```suggestion
import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
```

Reply:
With the suggested change you would have to change the code below, alternatively you can do,
```
import canUseTouchScreen from '../../libs/DeviceCapabilities/canUseTouchScreen';
```
and that's fine too.